docs/mining/how-to-stake.md

```diff
@@ -1,37 +1,40 @@
-# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> How to Stake
+# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> How to Stake/Vote
+
+Last updated for v1.3.0
 
 ---
 
-## How to Stake
+## How to Stake/Vote
 
-Participating in proof-of-stake requires a Decred wallet to be running 24/7. The wallet needs to be always online so that it can be called to vote - if the wallet is unavailable then the votes will be missed and no block reward will be received.
+Participating in Proof-of-Stake (PoS) voting requires tickets to be controlled by a Decred wallet running 24/7. The wallet needs to be always online so that it can respond by signing the block when its tickets are called to vote; if the wallet is unavailable then the votes will be missed and no block reward will be received.
```

Comment:
Neither "tickets controlled by a Decred wallet" or "signing the block" seem correct here. The original sentence made more sense, especially for a surface level overview of the process.

Reply:
Have reverted back to the original sentence, as we know that is correct and I think keeping it high-level here is fine. @RichardRed0x , if you want to defend/refine updated wording, can put back in.

## How to Stake/Vote

Participating in Proof-of-Stake (PoS) voting requires a Decred wallet to be running 24/7. The wallet needs to be always online so that it can be called to vote; if the wallet is unavailable then the votes will be missed and no block reward will be received.